crates/scmi/src/main.rs

```diff
@@ -0,0 +1,182 @@
+// SPDX-FileCopyrightText: Red Hat, Inc.
+// SPDX-License-Identifier: Apache-2.0
+// Based on implementation of other devices here, Copyright by Linaro Ltd.
+
+//! vhost-user daemon implementation for
+//! [System Control and Management Interface](https://developer.arm.com/Architectures/System%20Control%20and%20Management%20Interface)
+//! (SCMI).
+//!
+//! Currently, the mandatory parts of the following SCMI protocols are implemented:
+//!
+//! - base
+//! - sensor management
+//!
+//! As for sensor management, support for industrial I/O (IIO) Linux devices
+//! and a fake sensor device is implemented.
+//!
+//! The daemon listens on a socket that is specified using `--socket-path`
+//! command line option.  Usually at least one exposed device is specified,
```

Comment:
What happens if we don't specify any device?

Reply:
There will be no sensor device available in the guest OS but VirtIO SCMI will be still working there (like querying general info, available protocols, available devices, etc.).

Reply:
What about mentioning that at least in the README ?

Reply:
Done.